### DISCUSSION TITLE
Should we rename the Torrust projects?

### DISCUSSION THREAD

**Discussion by mickvandijke** [torrust/torrust-index-archive#17] [Announcements]
Hey guys,

Soon we want to start promoting the Torrust project to reach new users/contributers, but I was thinking that the naming of our projects might be a little confusing. 

So I have been toying with the idea of renaming the [Torrust Tracker](https://github.com/torrust/torrust-tracker) to just "Torrust", since the tracker is our main center of attention and most used of the current two projects. And also rename the [Torrust Index](https://github.com/torrust/torrust) to either "Torvue" (after the Vue framework we use) or "Nautilus" (after the company that started this project: [Nautilus Cyberneering GmbH](https://nautilus-cyberneering.de/)). This would make it easier to distinguish the two projects in my opinion.

I'd love to hear your input on this! And if you guys have any other naming ideas, please comment 😄

**Comment by BelieveInBunny** [comment]
Torrust is a good name and I really like it. Regarding torvue, somehow it feel kind of off to me (person opinion) and for Nautilus, it might be risky to use in case there is problem with them or project. it could damage/improve the image of the project.

I personally like below name, you can see 
- Tordeus
- Torran
- Tor0 (Torzero)
- ANx.


Thank you

**Comment by ldpr** [comment]
For what it's worth, I think nautilus is a great name and if you can use it, would be better than Torvue. 

It's nice to have a throwback to the company that support this project also.

**Reply by BelieveInBunny** [reply]
it is true and I also like it but I hope that company has no issue with this decision as I have seen some companies which prefer not to get their name used.

**Comment by da2ce7** [comment]
- https://github.com/orgs/torrust/discussions/13
